src/lib/hooks/useServiceWorker.js

```diff
@@ -46,5 +48,6 @@ export default () => {
       }
     }
     setInitFunctions(store.set('wallet'), store.set('userStorage'))
-  })
+    return () => interval.current && clearInterval(interval.current)
```

Comment:
there's exists a ready-to-use useInterval hook: https://github.com/donavon/use-interval/blob/master/src/index.js
you could:

```
import useInterval from 'use-interval'
// ...

const regRef = useRef()

useInterval(() => regRef.current && regRef.current.update(), 3600000)

// ...
       const onRegister = reg => {
         // force check for service worker update
         reg.update()
         regRef.current = reg

         if (reg.waiting) {
           onUpdate(reg)
         }
       }
// .. line 51 (on mount effect cancellation)
return () => regRef.current && regRef.current = null
```